packages/pyright-internal/src/tests/lsp/languageServerTestUtils.ts

```diff
@@ -211,7 +211,7 @@ export function createFileSystem(projectRoot: string, testData: FourSlashData, o
         mountedPaths.set(bundledStubsFolder, bundledStubsFolderPathTestServer);
     }
 
-    const vfsInfo = createVfsInfoFromFourSlashData(projectRoot, testData);
+    const vfsInfo = createVfsInfoFromFourSlashData(projectRoot, testData, /* ignoreRawConfigJson */ true);
```

Comment:
This change makes it possible to read the pyrightconfig.json as a file instead of having it be part of the vfsInfo.